services/collaboration/pkg/command/server.go

```diff
@@ -44,17 +46,24 @@ func Server(cfg *config.Config) *cli.Command {
 				return err
 			}
 
-			gwc, err := helpers.GetCS3apiClient(cfg, false)
```

Comment:
AFAICS `GetCS3apiClient` is unused now. So IMO we should remove it from the helpers module

Reply:
thanks, fixed.